spec/forms/idv/api_image_upload_form_spec.rb

```diff
@@ -300,6 +303,36 @@
         expect(response.errors).to have_key(:front)
         expect(response.errors).to have_value([I18n.t('doc_auth.errors.doc.resubmit_failed_image')])
       end
+
+      it 'logs analytics event' do
+        form.submit
+        expect(fake_analytics).to have_logged_event(
```

Comment:
alot of the values here are nil which is not  ... is it possible we could use _hash_including_ instead?